remotes/docker/fetcher.go

```diff
@@ -78,11 +78,8 @@ func (r dockerFetcher) Fetch(ctx context.Context, desc ocispec.Descriptor) (io.R
 
 			rc, err := r.open(ctx, req, desc.MediaType, offset)
 			if err != nil {
-				if errdefs.IsNotFound(err) {
-					continue // try one of the other urls.
-				}
-
-				return nil, err
+				log.G(ctx).WithError(err).Debug("failed to open remote reader")
```

Comment:
The terminology used here is inconsistent. I see "external url" , "alternative url", and this just generically refers to it as "remote". Since it is previously mentioned in the log as "alternative url", this messaged should probably say "failed to open alternative url", or change the previous log to "external url". Not sure we ever put a good name on this horrible feature :)

Reply:
Thank you for the review. Fixed.